zipkin-lens/src/components/App/Sidebar.test.js

```diff
@@ -39,12 +39,14 @@ describe('<Sidebar />', () => {
   it('should render internal links', () => {
     const wrapper = shallow(<Sidebar />);
     const list = wrapper.find('[data-testid="internal-links"]');
+    // Only discover page is the internal link.
```

Comment:
I guess I understand now that "internal link" means link served by the application here (routed in javascript) as opposed to "external link" meaning http request.. is that right?